.env

```diff
@@ -12,7 +12,7 @@ APP_SECRET=EDITME
 # Format described at http://docs.doctrine-project.org/projects/doctrine-dbal/en/latest/reference/configuration.html#connecting-using-a-url
 # For a sqlite database, use: "sqlite:///%kernel.project_dir%/var/data.db"
 # Set "serverVersion" to your server version to avoid edge-case exceptions and extra database calls
-DATABASE_URL=mysql://root@127.0.0.1/sylius_%kernel.environment%
+DATABASE_URL=mysql://sylius:nopassword@mysql/sylius
```

Comment:
Hi @tonicospinelli , may I ask why would you want to change it?

Reply:
I changed it to give a smooth experience for new adopters because the docker is not parameterized for multi-environment yet.